tests/test_playbooks/tasks/role.yml

```diff
@@ -17,6 +17,11 @@
     description: "{{ role_description }}"
     locations: "{{ role_locations }}"
     organizations: "{{ role_organizations }}"
+    filters:
```

Comment:
this makes the tests create the role *always* with those filters and we can't test idempotentcy well.

could you make it `filters: "{{ role_filters | default(omit) }}" here and pass `role_filters` as a var in `test_playbooks/role.yml` in some (but not all) cases? something like the following workflow should be tested: create a role without a filter, ensure that creating it again without a filter does not yield any changes, add a filter (expect changes), add the same filter (no changes), remove filter (changes).